common/djangoapps/student/helpers.py

```diff
@@ -268,14 +268,22 @@ def get_next_url_for_login_page(request):
                 # Tries reversing the LMS dashboard if the url doesn't exist
                 redirect_to = reverse('dashboard')
 
+            if redirect_to_logistration:
+                redirect_to = settings.LMS_ROOT_URL + reverse('dashboard')
```

Comment:
I believe we should get `LMS_ROOT_URL`  from `configuration_helpers` and default it to `settings.LMS_ROOT_URL`
```
configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL)
```